src/App.tsx

```diff
@@ -13,7 +11,7 @@ const App: React.FC = () => {
         fileType={FileType.csv}
         onChange={function() {}}
       />
-      <TimelineComponent data={new Data('path/to/file', [{}])} />
+      {/* <TimelineComponent data={new Data('path/to/file', [{}])} />*/}
```

Comment:
Why did you comment out the TimelineComponent?

Reply:
The TimelineComponent is rendered in the ParseComponent, so it should not be in the App.tsx file.